docs/native-modules-setup.md

```diff
@@ -5,22 +5,26 @@ title: Native Modules Setup
 
 Native modules are usually distributed as npm packages, except that on top of the usual Javascript they will include some native code per platform. To understand more about npm packages you may find [this guide](https://docs.npmjs.com/getting-started/publishing-npm-packages) useful.
 
-To get set up with the basic project structure for a native module we will use a third party tool [create-react-native-module](https://github.com/brodybits/create-react-native-module). You can go ahead further and dive deep into how that library works, for our needs we will only need:
+To get set up with the basic project structure for a native module we will use the community tool [@react-native-community/bob](https://github.com/react-native-community/bob)\**. You can go ahead further and dive deep into how that library works, for our needs we will only need:
```

Comment:
Can you put the note about `create-react-native-module` in the quote block (`>`) at the bottom of the section instead of using annotation? Example: https://reactnative.dev/docs/getting-started#interactive-examples

Reply:
Done

Reply:
Still `**` asterix characters are present in the content.

Reply:
I have fixed that :)